tests/toolbox_image.sh

```diff
@@ -13,6 +24,55 @@ result=0
 
 exit_on_failure=false
 
+format_dummy_partition()
+{
+    command="${1}"
+    sector_start="${2}"
+    device_file="${3}"
+
+    loop_device="$(chroot "${rootfs_dir}" /sbin/losetup -f)"
+    echo "Using next available loop device '${loop_device}' to format partition using: '${command}':"
+
+    touch "${rootfs_dir}${loop_device}"
+    mount --bind "${loop_device}" "${rootfs_dir}${loop_device}"
+
+    chroot "${rootfs_dir}" /sbin/losetup -o "${sector_start}" "${loop_device}" "${device_file}"
+    chroot "${rootfs_dir}" "${command}" "${loop_device}"
+    chroot "${rootfs_dir}" /sbin/losetup -d "${loop_device}"
+
+    if grep "${loop_device}" "/proc/mounts"; then
+        umount "${rootfs_dir}${loop_device}"
+        unlink "${rootfs_dir}${loop_device}"
+    fi
+}
+
+create_dummy_storage_device()
+{
+    partition_table_file="${rootfs_dir}/tmp/partition_table"
```

Comment:
why store this in a file, if it is created AND removed in this function only. Just store the data in a variable and use the variable as stdin if possible. Unless it is really needed avoid storing temporary data in files

Reply:
See comment down the line, i don't see how it is possible using chroot